Description: Debian has buggy gtk headers, so we have to disable strict checking
 Author: Antoine Martin <antoine@devloop.org.uk>

Index: setup.py
--- a/setup.py	2015-08-08 22:20:57.622648693 +0700
+++ b/setup.py	2015-08-08 22:21:30.200046560 +0700
@@ -615,7 +615,7 @@
                         #"-Wno-error=uninitialized",
                         #needed on Debian and Ubuntu to avoid this error:
                         #/usr/include/gtk-2.0/gtk/gtkitemfactory.h:47:1: error: function declaration isn't a prototype [-Werror=strict-prototypes]
-                        #"-Wno-error=strict-prototypes",
+                        "-Wno-error=strict-prototypes",
                         ]
             else:
                 #older versions of OSX ship an old gcc,
